src/components/bcros/filing/common/filedAnd/Pending.vue

```diff
@@ -0,0 +1,16 @@
+<template>
+  <div>
+    <UIcon name="i-mdi-clock-outline" class="text-lg text-orange-500 mr-1 -mb-1" />
+    <span>{{ $t('text.filing.pending') }}</span>
+    <BcrosDivider class="mx-2" />
+    <span>{{ $t('text.filing.paymentCompleted') }}</span>
+  </div>
+</template>
+
+<script setup lang="ts">
+import type { ApiResponseFilingI } from '#imports'
+
+defineProps({
+  filing: { type: Object as PropType<ApiResponseFilingI>, required: true }
+})
+</script>
```

Comment:
Looks like this:

<img width="912" height="182" alt="image" src="https://github.com/user-attachments/assets/d64bdcce-7843-4fae-b67c-bd18522af9c0" />